textbook/08/1/Libraries.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>Note: This cell is hidden (collapsed) in the rendered HTML thanks to the cell tag <code>hide_cell</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Well OK that's how ReviewNB works.

Anyway, yes, all those verbose cells are hidden by default.

The problem is that the Notebooks are meant to be downloadable and interacted with by readers; but, there's not yet any real handling for separate data files. There are various ways of handling this; for example, we can load files from a/the server via URL. But, it's useful for the whole process to keep things all together. (It's also not such a bad thing for readers to see the plain text of the data they're interacting with.)

![image](https://user-images.githubusercontent.com/530998/195705868-d71a6109-480b-4535-abc3-c0ccfcaa8e18.png)